ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# ISSUE
+
+## Reproduction Steps
+
```

Comment:
What about adding here:
- `Expected behaviour`
- `Observable behaviour`
- `Screenshots` (if applicable)

what do you think?